src/server/game/Battlegrounds/BattlegroundMgr.h

```diff
@@ -140,8 +140,8 @@ class BattlegroundMgr
     BattlegroundQueue m_BattlegroundQueues[MAX_BATTLEGROUND_QUEUE_TYPES];
 
     std::vector<uint64> m_ArenaQueueUpdateScheduler;
-    bool   m_ArenaTesting;
-    bool   m_Testing;
+    bool   m_ArenaTesting = sWorld->getBoolConfig(CONFIG_DEBUG_ARENA);
+    bool   m_Testing = sWorld->getBoolConfig(CONFIG_DEBUG_BATTLEGROUND);
```

Comment:
aren't these deprecated now? cc @Winfidonarleyan 

also I'm not sure about this